main.go

```diff
@@ -238,7 +241,8 @@ func argParse() {
 		"The differ to be run with debug log level and the SDK/gocb logging will also be enabled.")
 	flag.IntVar(&options.setupTimeout, "setupTimeout", base.SetupTimeoutSeconds,
 		"Common setup timeout duration in seconds")
-
+	flag.StringVar(&options.xattrsNoCompare, "xattrsNoCompare", "",
+		"A string denoting the Xattrs that shouldn't be considered for comparison, The induvidual Xattrs are seperated by \":\"")
```

Comment:
I think i prefer the second method over the first, because in future if there is another system Xattr that needs to be excluded that differ isn't aware of, then the second method comes handy.